 Update main.c

Removed bug where cron.update is not picked up while jobs are still running. (We have long running cronjobs and experienced long overdue cron.update files)

diff --git a/main.c b/main.c
index 595f07a..0eba280 100644
--- a/main.c
+++ b/main.c
@@ -338,7 +338,8 @@ main(int ac, char **av)
 					SynchronizeDir(SCDir, "root", 0);
 					ReadTimestamps(NULL);
 				}
-			} else {
+			} 
+			if (rescan < 60) {
 				CheckUpdates(CDir, NULL, t1, t2);
 				CheckUpdates(SCDir, "root", t1, t2);
 			}

